public/data/freedompraise.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name":"Praise Dike",
+  "type":"personal",
+  "bio":"Open Source Enthusiast",
+  "avatar":
```

Comment:
you can remove this avatar field from here, and you must have links field in your profile for it to work